packages/is-shallow-equal/.npmignore

```diff
@@ -0,0 +1,2 @@
+benchmark
+test
```

Comment:
I'd much rather have kept the whitelist and updated it per the new file structure, than to break convention (and [common recommendations](https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d)) to resort to the blacklist.

Reply:
4ed5b58 addresses this.